templates/404.php

```diff
@@ -10,7 +10,7 @@
 	$title = '404 - Not Found';
 }
 
-WPUtil\Component::render('components/banners/banner-default', [
+get_template_part('components/banners/banner', 'default', [
 	'title' => $title
```

Comment:
@gravcmcguire use this to grab the title override, the banner component is no longer accounting for this.

`'title' => trim(get_field('banner_title_override', $post_id)) ?: get_the_title(),`